src/styles/_langselector.scss

```diff
@@ -69,24 +69,31 @@ $border-col: rgba(229,231,235, 1);
     color: $black;
     margin: 4px 0px;
     margin-top: 0px;
-    padding: 4px 16px;
+    padding: 4px 20px;
     font-size: 15px;
     font-family: 'Roboto', 'Helvetica Neue', Helvetica, Arial, sans-serif;
     line-height: 18px;
     &.selected {
-      background: #F3F3F3;
+      // background: #F3F3F3;
       @media (max-width: $screen-xs-max) {
         font-weight: bold;
       }
     }
 
     &:hover {
-      color: $btn-hover-blue;
+      background: #F3F3F3;
```

Comment:
Should we possibily share this background color?
```$nav-bg: #F3F3F3 !default;```

Reply:
good call!